specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json

```diff
@@ -158,6 +161,22 @@
           "description": "the disk sku.",
           "$ref": "Disks.json#/definitions/AccountType",
           "readOnly": true
+        },
+        "creationSourceUri": {
```

Comment:
Please note adding new properties to existing API version is considered as a breaking change as the previously generated SDKs will not have these properties. Ideally need to introduce a new API version. 
I would recommend to expose a new API version as the current API version exists from a long time 2018-03-01-preview and kind of qualify as a GA version for customers.

Reply:
Please refer https://aka.ms/bcforapi

Reply:
sure, we will introduce a new api version for this changes.
@FireDefend  FYI.